plugins/thunar-apr/thunar-apr-image-page.c

```diff
@@ -173,7 +172,9 @@ thunar_apr_image_page_init (ThunarAprImagePage *image_page)
   /* add labels for the Exif info */
   for (n = 0; n < G_N_ELEMENTS (TAIP_EXIF); ++n)
     {
-      label = gtk_label_new (_(TAIP_EXIF[n].name));
+      const gchar *exif_title = exif_tag_get_title ( TAIP_EXIF[n] );
```

Comment:
I'm not a guru or whatever, but if the titles are already translated in thunar's translation, even if it's duplication of work - then more languages will be covered by what we have compared to what libexif would provide ?

As for 1, whichever works - would be better to avoid deprecated APIs - ask upstream why they want ppl to use the _ifd version instead of the (seemingly) simpler one ?